store-front/src/design-system/molecules/NftCard/NftCard.tsx

```diff
@@ -336,7 +336,7 @@ export const NftCard: React.FC<NftCardProps> = ({ loading, ...props }) => {
                             launchTime > 0 &&
                             `${new Date(launchTime).getDate() - 1} day${new Date(launchTime).getDate() > 2 ? 's' : ''
                             } - ${format(
-                                new Date(launchTime),
+                                new Date(launchTime + new Date().getTimezoneOffset() * 60 * 1000),
```

Comment:
Are you sure this is the solution and not a `minus timezoneoffset`? Because I think what I saw was a drop date that was 1 hour too much. We're in +1 timezone, so this code change would intuitively increase the wrong offset to 2 hours too much